zerver/lib/avatar.py

```diff
@@ -27,25 +27,30 @@ def user_avatar_hash(email):
     user_key = email.lower() + settings.AVATAR_SALT
     return make_safe_digest(user_key, hashlib.sha1)
 
-def avatar_url(user_profile):
+def avatar_url(user_profile, medium=False):
     # type: (UserProfile) -> text_type
     return get_avatar_url(
             user_profile.avatar_source,
-            user_profile.email
+            user_profile.email,
+            medium
     )
 
-def get_avatar_url(avatar_source, email):
+def get_avatar_url(avatar_source, email, medium=False):
     # type: (text_type, text_type) -> text_type
+
+    medium_gravitar = "&s=500" if medium else ""
```

Comment:
It might be more clear to name these variables as `gravitar_suffix` and `suffix`, so they read less confusingly for the medium=False case.